src/admin-views/commerce/orders/single/order-items-coupons.php

```diff
@@ -24,7 +26,7 @@
 		<td><?php echo esc_html( $coupon['display_name'] ); ?></td>
 		<td><code><?php echo esc_html( $coupon['slug'] ); ?></code></td>
 		<td class="tec-tickets-commerce-single-order--items--table--row--info-column"></td>
-		<td style="padding-left:0;">
+		<td style="padding-left: 0;">
```

Comment:
Can we do this without inline styles?

Reply:
I was following the convention from other templates, but I can change to use a stylesheet.